src/utilities/generic.ts

```diff
@@ -132,6 +132,13 @@ export function isEmpty(value: any): boolean {
   return value === null || value === undefined || value === '' || Number.isNaN(value);
 }
 
+/**
+ * Returns true if `value` is neither null nor undefined, else returns false.
+ */
+export function isNullOrUndefined(value: unknown): value is null | undefined {
```

Comment:
The way I've been doing this is is by doing `value == null`. The double equals applies for both `null` and `undefined`. It is a less explicit than this though. I can go either way.

If you do decide to keep this function, do you mind adding a few quick unit tests for it?

Reply:
Think your solution is cleaner. Left a comment to explain the check.